test/server-unit/db.spec.js

```diff
@@ -0,0 +1,43 @@
+const db = require('../../server/lib/db');
+const { expect } = require('chai');
+var _ = require('lodash');
+
+var dbTest = function () {
+
+  before(function () {
+    var params = {
+      host     : 'localhost',
+      user     : 'bhima',
+      password : 'HISCongo2013',
+      database : 'bhima_test',
+    };
+    db.init(params);
+
+  });
+
+  it('Should check the connection to mysql', function (done) {
+    var fx = db.pool.getConnection(function (err, result) {
+      if (err) {
+        done(err);
+        return;
+      }
+      expect(true).to.equal(true);
+      done();
+    });
+
+  });
+
+  it('#Should try to retrieve data from a specific table (unit)', () => {
+    return db.exec('SELECT * FROM unit', {})
+      .then((rows) => {
+				const test = rows.length > 0;
+        expect(test).to.equal(true);
+      })
+      .catch((error) => {
+        return error;
```

Comment:
This code will ignore the error, but we want tests to fail if there is an error that occurs.  It looks like [mocha supports promise errors](http://tobyho.com/2015/12/16/mocha-with-promises/), so you can just remove the `.catch()` statement and it should handle the error automatically.